Apps/Common/src/Models/NotificationSettingsBase.cs

```diff
@@ -38,6 +38,7 @@ public NotificationSettingsBase()
         /// <param name="notificationSettings">Initialize values from passed in object.</param>
         public NotificationSettingsBase(NotificationSettingsBase notificationSettings)
         {
+            this.SubjectHdid = notificationSettings.SubjectHdid;
```

Comment:
This needs to be moved to NotificationSettingsRequest as it is specific to the request object.  Unless you convince Tyler to echo it back :).

Reply:
we need hdid in the response to correlate or the response can't really be meaningfully processed in a stateless manner. I sent Tyler the change that we will be putting hdid in request and response payload. If there is a reason that's hardship we can move the hdid to the Request  only.

Reply:
Can I get consensus on this? Would we ever need to process the response in the context of the hdid outside of the synchronous call? If not, I can put hdid only in request.  I did mention to Tyler we'd put it in both request and response.